tasks.php

```diff
@@ -1177,7 +1177,7 @@ function TaskForm($task)
     echo "<td colspan='2'>";
     echo "<b>" . property_get_label('task_summary', FALSE) . "&nbsp;</b>";
     echo "&nbsp;&nbsp;";
-    echo "<input type='text' name='task_summary' value=\"$task_summary_enc\" size='60' maxlength='80'>";
+    echo "<input type='text' name='task_summary' value=\"$task_summary_enc\" size='60' maxlength='80' required>";
```

Comment:
Might be a good idea to make `task_details` required as well, since not providing it shows this:
 
![image](https://user-images.githubusercontent.com/3049847/52264277-84396100-2931-11e9-9fb8-c245ea40be2b.png)